### ISSUE TITLE
Update 1.3.15

### ISSUE THREAD

**Issue by TokyoSU** [MontyTRC89/Tomb-Editor-Public#2]
This repository need to be updated to 1.3.15.
At last without TEN code if it's present.

**Comment by Lwmte** [comment]
I agree, same was mentioned in #1 , we need to somehow keep public repo up to date with current development status @MontyTRC89
